tests/testthat/test-exclude-potential-pcos.R

```diff
@@ -1,119 +1,24 @@
-bef <- tibble::tribble(
-  ~koen, ~pnr, ~foed_dato,
-  # Men (as defined by PNR)
-  1, 1000000001, "1980-01-01",
-  # Women (as defined by PNR)
-  2, 2000000000, "1980-01-01",
-  # No GLD purchases (excluded)
-  1, 3000000001, "1980-01-01",
-  2, 4000000000, "1980-01-01",
-) |>
-  dplyr::mutate(
-    pnr = as.character(pnr),
-    foed_dato = lubridate::as_date(foed_dato)
+bef <- simulate_registers("bef", 1000)[[1]] |>
+  dplyr::add_row(
+    koen = 2,
+    foed_dato = "20000101"
+  )
+gld_purchases <- simulate_registers("lmdb", 1000)[[1]] |>
+  include_gld_purchases() |>
+  dplyr::add_row(
+    date = "20200101",
+    atc = "A10BA02",
+    # Not the indication code
```

Comment:
```suggestion
    # Not the indication code.
```

Wait, what does this comment mean?

Reply:
I updated your suggestion.